tests/test_brew_collector.py

```diff
@@ -611,13 +611,18 @@ def test_fetch_rpm_build(mock_sca):
     ]:
         assert package in provides
     assert len(srpm.get_upstreams()) == 1
```

Comment:
get_upstreams() also uses get_ancestors(include_self=True). I wonder if that should be changed as well (but doesn't have to be in this PR necessarily).

Reply:
That one looks a bit more tricky because it inherits that behavior from `get_roots()` and I'm a bit wary of changing that one. I'll file an issue to investigate this a bit more in-depth.